Automattic/wp-calypso
# Theme Showcase: Revamp header

#### Proposed Changes

This PR updates the UI of (1) the `<ThemesHeader />` component, and (2) the page layout's background and padding. The design spec can be found in 0EogtxvlmxMGcnEEaMVECv-fi-1508%3A52993. See the comparison:

| Before | After | After v2 |
| --- | --- | --- | 
| ![Screen Shot 2022-11-19 at 11 34 33 PM](https://user-images.githubusercontent.com/797888/202858682-b6ea31df-e00c-47de-837c-abc3ad016530.png) | ![Screen Shot 2022-11-19 at 11 34 51 PM](https://user-images.githubusercontent.com/797888/202858697-0897b967-d01f-4cc9-919a-ac28f82c9195.png) | ![Screen Shot 2022-11-23 at 3 22 23 PM](https://user-images.githubusercontent.com/797888/203491296-e53b716a-fac7-43e1-a8b1-10e450456ea1.png)


IMPORTANT NOTES:
1. The header and search components do not sticky with user scroll. @SaxonF I remember this was the conclusion for the time being, CMIIW 🙂
2. @SaxonF we need to address the screen option tab, which was not considered in the design spec. Unfortunately, this component will overlap with the "Install theme" button, which is why I had to add extra top padding to the header (IMO doesn't look very clean). For reference, this is the screen option tab.
![Screen Shot 2022-11-19 at 11 44 14 PM](https://user-images.githubusercontent.com/797888/202859174-11f37701-2b9d-4ee3-a13c-6f936daa5c26.png). 
3. I discovered that the Calypso dashboard uses very the breakpoint 660px instead of Gutenberg's 600px. The usage of 660px is across main layouts, which might be worth updating in the future, but it's out of scope for this PR. In the meantime, this PR will still use the 660px breakpoint via `breakpoint-deprecated( ">660px" )`.

#### Testing Instructions



* Head to the Theme Showcase `/themes/${site_slug}`.
  * If using calypso.live, the flag themes/showcase-i4/search-and-filter is required.
* Ensure that the PR updates the page UI according to the design spec.
* Ensure that mobile/table resolution works well with the new UI.
* Ensure that the existing UI works as before.

#### Pre-merge Checklist
